code/game/objects/random/random.dm

```diff
@@ -875,23 +875,32 @@ obj/random/obstruction/item_to_spawn()
 	desc = "LOADSAMONEY!"
 	icon = 'icons/obj/items.dmi'
 	icon_state = "spacecash1"
+	spawn_nothing_percentage = 75
 
 /obj/random/cash/item_to_spawn()
-	return pickweight(list(/obj/item/weapon/spacecash/ewallet/random/c200 = 8,
+	return pickweight(list(
+				/obj/item/weapon/spacecash/ewallet/random/c50 = 24,
+				/obj/item/weapon/spacecash/ewallet/random/c100 = 16,
+				/obj/item/weapon/spacecash/ewallet/random/c200 = 8,
 				/obj/item/weapon/spacecash/ewallet/random/c500 = 6,
-				/obj/item/weapon/spacecash/ewallet/random/c1000 = 4,
-				/obj/item/weapon/spacecash/ewallet/random/c5000 = 2,
-				/obj/item/weapon/spacecash/ewallet/random/c10000 = 1))
+				/obj/item/weapon/spacecash/ewallet/random/c1000 = 4))
```

Comment:
I wouldn't remove the high spawns, just make them rarer. You can use decimal values here. Let people experience excitement for finding a big chunk of cash, just make it so it can't happen every single round through unfavorable RNG.

Reply:
High spawns have been removed at request of West. Resolved.